app/src/renderer/components/staking/PageStaking.vue

```diff
@@ -7,12 +7,40 @@ tm-page(title='Staking')
       i.search.material-icons search
 
   modal-search(type="delegates" v-if="somethingToSearch")
+
+  //- Tabs
   .delegates-tabs
     .tab(v-for="(tab, i) in tabs",
       :key="'tab-' + i",
       :class="{'tab-selected': i === tabIndex}",
-      @click="tabIndex = 1") {{tab}}
-  .delegates-container
+      @click="tabIndex = i") {{tab}}
+
+  //- My Stake
+  template(v-if="tabIndex === 0")
```

Comment:
do we want to implement tabs this way? i think we should use vue-router to switch routes.

Reply:
OK, I'll look into it.

Reply:
I implemented it with Vue Router and don't like it because if I selected the `Validators` tab, then click to `Wallet`, then click back to `Staking`, I would expect the `Validators` tab to remain open.  If we use routes then it will always open `My Stake` when going back to `Staking`.

Also, pressing the Back button doesn't update `tabIndex` so we'd have to solve that somehow.

@jbibla, what do you think?

Reply:
i don't think that's the expected behaviour. 

staking is the parent route but it only displays the children. therefore "staking" should always take the user to "my stake".

i guess the back button can be addressed by checking the `$route` and setting it accordingly.

Reply:
OK, then assuming I can fix the back button I'll keep the routed implementation.

Reply:
Done using your suggestions!